pkg/helm/helm.go

```diff
@@ -115,61 +104,23 @@ func (h *Hydrator) templateArgs(ctx context.Context, destDir string) ([]string,
 }
 
 func (h *Hydrator) appendValuesArgs(args []string) ([]string, error) {
-	switch h.ValuesFileApplyStrategy {
-
-	case "", ValuesFileApplyStrategyOverride:
-		for _, vs := range h.ValuesFilePaths {
-			if vs == "" {
-				continue
-			}
-			args = append(args, "--values", vs)
+	for _, vs := range h.ValuesFilePaths {
+		if vs == "" {
```

Comment:
seems like this should be validated in to disallow empty paths. silently skipping them probably indicates a problem

Reply:
sg, returning an error now